.github/workflows/nightly-release.yml

```diff
@@ -0,0 +1,25 @@
+name: nightly-release
+
+on:
+  schedule:
+    # 13:00 UTC is 05:00 in Pacific standard time (UTC-8), which is well
+    # after nightly TensorFlow wheels are released (around 2--3 AM).
+    # (cron syntax: minute hour day-of-month month day-of-week)
+    - cron: '0 13 * * *'
+
+jobs:
+  ci:
+    uses: ./.github/workflows/ci.yml
+
+  nightly-release:
+    runs-on: ubuntu-18.04
+    needs: ci
+    if: github.repository == 'tensorflow/tensorboard'
+    steps:
+      - name: Publish the package
+        run: |
+          twine check /tmp/tb_nightly_pip_package/tb_nightly*.whl
+          twine upload -r testpypi /tmp/tb_nightly_pip_package/tb_nightly*.whl
```

Comment:
Optional but maybe add a TODO here to switch off of testpypi, just for posterity to indicate this is temporary